src/blueprints/tv2/value-objects/tv2-action.ts

```diff
@@ -0,0 +1,46 @@
+import { PartAction, PieceAction } from '../../../model/entities/action'
+
+export enum Tv2ActionContentType {
```

Comment:
Are these going to be one-to-one with `PieceType`?

Reply:
Most likely.